gt/main.c

```diff
@@ -900,6 +1133,24 @@ gt_proc(void *arg)
 			ETHER_TYPE_IPv4);
 		process_pkts_acl(&gt_conf->net->front, lcore, &acl6,
 			ETHER_TYPE_IPv6);
+
+		if (cur_tsc - last_tsc >= frag_scan_timeout_cycles) {
+			death_row.cnt = 0;
```

Comment:
If there's no bug in the code, `death_row.cnt` must be zero here. So we can turn this assignment into an `RTE_VERIFY(death_row.cnt == 0);`.